pandas/tests/extension/array_with_attr/array.py

```diff
@@ -49,6 +49,7 @@ def __init__(self, values, attr=None) -> None:
 
     @classmethod
     def _from_sequence(cls, scalars, *, dtype=None, copy=False):
+        copy = np._CopyMode.ALWAYS if copy else np._CopyMode.IF_NEEDED
```

Comment:
Will these `_CopyMode`'s always be private?

Reply:
There are no plans for making these public. If using private members doesn't fit here then there can be a `if` checking NumPy version.

Reply:
Ya that would be preferable if possible. pandas would also like to not use private numpy APIs for numpy's sake :)